upgrade/helpers/rhevm.py

```diff
@@ -262,14 +262,14 @@ def validate_and_create_product_templates():
     if sat_instance and cap_instance:
         sat_host = os.environ.get('RHEV_SAT_HOST')
         new_sat_template = os.environ.get('RHEV_SAT_IMAGE') + "_new"
-        cap_host = os.environ.get('RHEV_CAP_HOST')
+        cap_hosts = [os.environ.get('RHEV_CAP_HOST')]
```

Comment:
This will fail if user specified two capsules, as RHEV_CAP_HOST returns a string and you cannot directly covert it into list. Like this:
```
>>> cap_hosts = 'something1.sample.com, someting2.sample.com'
>>> cap_hosts = [cap_hosts]
>>> cap_hosts
['something1.sample.com, someting2.sample.com']
```
That becomes a single item in list and not two

Reply:
RHEV_CAP_HOST , is internal variable , when user doesn't specify anything. This task is for creation of internal templates only

Reply:
But what if we start using two capsules internally?

Reply:
For RHEV based Capsule, to create a template, provide a provision for single capsule. That way it will eliminate the 'cap_hosts' var and use 'cap_host'.